src/backend/base/langflow/components/vectorstores/AstraDB.py

```diff
@@ -110,12 +157,6 @@ class AstraVectorStoreComponent(LCVectorStoreComponent):
             info="Optional list of metadata fields to include in the indexing.",
             advanced=True,
         ),
-        HandleInput(
```

Comment:
@nicoloboschi definitely see your point here - but, i did maintain the embeddings support... the broken compatibility would be if someone had a flow that was using the AstraVectorize component as input to this input, right? in this PR, i remove that component entirely since its built in now... are you suggesting we should keep the separate component as well, for backwards compatibility purposes?

I *think* in other cases the backwards compatibility is maintained...

Reply:
I think that that was mentioned yeah, keep the separate component around for backwards compatibility. On the other hand...how many people have a flow using vectorize right now? If we can significantly reduce confusion for future vectorize users by removing this separate component, it may be worth breaking backwards-compatibility in this case. 

(disclaimer: I, of course, would never advocate for breaking backwards-compatibility)